apps/vercel/src/components/config-screen/ConfigScreen.styles.tsx

```diff
@@ -0,0 +1,34 @@
+import tokens from '@contentful/f36-tokens';
+import { css } from 'emotion';
+import { VercelBrand } from '../common/VercelIcon';
+
+export const styles = {
```

Comment:
Is this the right design for the background of the app config page? Maybe I'm looking at the wrong design file, but I think it was supposed to look more like this (no black background, no image at the bottom, etc.): 
<img width="681" alt="Screenshot 2024-01-23 at 4 38 27 PM" src="https://github.com/contentful/apps/assets/62958907/1f5266ee-7c8e-4a8a-933b-aab9e71bda22">

Reply:
We aren't currently doing an OATUH flow for this, so for consistency sake its following the previously established config screen designs (amplify) for base configuration since its not ready for prime time and designs aren't finalized